src/afError.js

```diff
@@ -4,10 +4,9 @@ angular.module('angularFormMessages').directive('afError', function () {
     require: '^afFieldWrap',
     scope: true,
     link: function linkFn($scope, elem, attrs, ctrl) {
-
-      $scope.$on('validation', function (event, modelPath, isValid, message) {
+      $scope.$on('validation', function (event, modelPath, errors) {
         if (modelPath === ctrl.modelPath) {
-          $scope.message = isValid ? undefined : message;
+          $scope.errors = errors;
```

Comment:
Ik zou er dan `messages` van maken, want het zou best kunnen dat we ook warnings en misschien zelfs success messages willen ondersteunen. Het sluit ook beter aan bij de naamgeving van `ng-messages`